spec/controllers/works_controller_spec.rb

```diff
@@ -1,87 +0,0 @@
-require 'spec_helper'
```

Comment:
It was testing the same things/ways as `spec/controllers/generic_works_controller_spec.rb` It seemed silly to have two.